### ISSUE TITLE
DeviceBrand unknown mobile class useragent list

### ISSUE THREAD

**Issue by cloverzrg** [nielsbasjes/yauaa#118]
```xml
<dependency>
            <groupId>nl.basjes.parse.useragent</groupId>
            <artifactId>yauaa</artifactId>
            <version>5.3</version>
</dependency>
```
```JavaScript
db.getCollection("ua").aggregate([
    {
        $match:{
            "DeviceBrand":"Unknown",
            "OperatingSystemClass":"Mobile",
        }
    },{
        $group:{
            _id:"$UserAgent",
            count:{$sum:1}
        }
    },{
        $sort:{count:-1}
    }
    ])
```

result
```json
[{
		"_id": "Mozilla/5.0 (Linux; Android 6.0.1; SM-J500M) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 918
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.0; LG-M250) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 816
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.0; SM-G570M) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 738
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.0.0; moto g(6) play) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 545
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.0; SM-G610M) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 491
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.1.0; Moto G (5S)) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 441
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0.1; SM-G532MT) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 440
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.1.0; Moto G (5S) Plus) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 353
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0.1; SM-J700M) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 350
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0; MotoG3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 337
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.0; Moto G (4)) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 320
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.1.1; Moto G Play) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 295
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.1.1; Moto E (4)) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 281
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 5.1.1; SM-G531H) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 271
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.0; LG-M320) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 264
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.1.1; Moto G (5S)) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 251
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.0; SM-J730G) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 227
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0; LG-X230) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 222
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0; LG-K430) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 218
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 5.1.1; SM-J105B) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 201
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 5.1.1; SM-J120H) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 201
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.0.0; moto e5 plus) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 198
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.0.0; moto g(6)) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 194
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.1.0; Moto G (5)) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 174
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.0; Moto C Plus) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 159
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.1.0; SM-J701MT) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 153
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.0; Moto G (5)) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 152
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.0.0; moto x4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 149
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0; XT1097) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 147
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.1.1; Moto E (4) Plus) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 146
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.0; SM-J710MN) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 144
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0; XT1069) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 136
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0.1; SM-G600FY) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 128
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.0.0; SM-J810M) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 126
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.1.0; SM-J530G) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 116
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 4.4.4; SM-G530H) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 115
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.0; Lenovo K53b36) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 112
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0; 5010E) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 111
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.0.0; moto g(6) plus) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 104
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.1.1; ASUS_X00LD) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 100
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 5.1.1; SM-J320M) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 97
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.1.1; SM-G611MT) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 93
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.0.0; moto e5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 92
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 5.1.1; ASUS_X013D) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 92
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0.1; SM-G900M) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 92
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 5.1.1; SM-J320M) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.64 Mobile Safari/537.36",
		"count": 85
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.0.0; SM-J600GT) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 82
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.0; SM-J530G) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 82
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.0; SM-A510M) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 81
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0.1; SM-G532M) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 75
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.0.0; SM-G955F) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 75
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0; XT1068) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 74
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 4.4.2; SM-G355M) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 73
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.1.0; motorola one) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 71
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 5.1.1; SM-G531BT) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 66
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 5.1.1; SM-J110M) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 66
	},
	{
		"_id": "Mozilla/5.0 (Android 7.1.1; Mobile; rv:62.0) Gecko/62.0 Firefox/62.0",
		"count": 65
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0; XT1078) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 65
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0.1; ASUS_A007) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 64
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 4.4.2; 4009E) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 62
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 4.3; SM-G7102T) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 61
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.0; MS50L) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 59
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0.1; E2363) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 58
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0.1; LG-K200) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 58
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.0.0; SM-G935F) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 58
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.0; Moto C) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 57
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0.1; Lenovo PB1-750M) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 55
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 5.0.2; XT1069) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 53
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.0.0; SM-G950F) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 52
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.1.1; SM-J510MN) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 51
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 5.0.2; SM-G530BT) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 50
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0.1; Lenovo K33b36) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 49
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.0; LG-M400) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 46
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0.1; SM-G532MT) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.64 Mobile Safari/537.36",
		"count": 46
	},
	{
		"_id": "Mozilla/5.0 (Android 7.0; Mobile; rv:63.0) Gecko/63.0 Firefox/63.0",
		"count": 45
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.1.0; Moto G (5) Plus) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 45
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.0.0; SM-G930F) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 45
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.1.0; Twist Metal 32GB) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 44
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.0.0; SM-A910F) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 44
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.0.0; ASUS_Z017DC) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 43
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0.1; LG-K220) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 43
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0; Z2 Play) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 42
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.0; Moto G (4)) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.64 Mobile Safari/537.36",
		"count": 42
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.0.0; XT1635-02) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 42
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 5.1; S420) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 42
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0; E5563) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 41
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 5.1; XT1033) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 41
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.0.0; SM-G570M) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 40
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0; MotoE2(4G-LTE)) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 40
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.1.0; ASUS_X00LD) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 39
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0.1; SM-G903M) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 39
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.0; Twist Max) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 36
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0; 5046J) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 36
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.1.0; moto e5 play) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 36
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0.1; SM-J500M) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.64 Mobile Safari/537.36",
		"count": 35
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 8.0.0; Moto Z2 Play) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 35
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 6.0.1; XT1225) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 34
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.1.1; SM-J250M) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 34
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 7.0; Moto G (5) Plus) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 32
	},
	{
		"_id": "Mozilla/5.0 (Linux; Android 5.0.2; LG-D337) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36",
		"count": 30
	}
]
```

**Comment by nielsbasjes** [comment]
Thanks for reporting this!
Turns out that with Chrome 70 on Android Google changed the pattern of the useragent string to no longer include the "BUILD/..." part.

Working on it.

**Comment by nielsbasjes** [comment]
I have fixed most of these.

**Comment by nielsbasjes** [comment]
I released version 5.4 with this high prio fix.
